draft-ietf-jsonpath-base.md

```diff
@@ -305,7 +305,7 @@ $.store.book[?(@.price < 10)].title
 | `@`         | the current node: within [filter selectors](#filter-selector)                                                                     |
 | `.name`     | child selectors for JSON objects: [dot selector](#dot-selector)                                                                     |
 | `['name']`  | child selectors for JSON objects: [index selector](#index-selector)                                                                 |
-| `..`        | descendants: [descendant selector](#descendant-selector)                                                                     |
+| `..name` <br> `..[3]` | descendants: [descendant selector](#descendant-selectors)                                                                     |
```

Comment:
> I'm not sure I agree with this being unified with the name/bracket notation. I get it, in that you can't just have `..` by itself, though. Maybe it's an implementation detail.

What spec change would you like to see? It would require an issue as the change is outside the scope of this PR.

> Secondly, _any_ bracket notation query will work: you could have `$..[?(@.foo)]` to recursively search for objects with `foo` properties.

The spec doesn't currently support filters as part of a descendant selector. Some implementations may support that. An issue would be needed as it's also outside the scope of this PR.

Reply:
> What spec change would you like to see?

I think what you have is fine to describe the syntax, but my implementation doesn't (and likely won't) actually do it that way.

> The spec doesn't currently support filters as part of a descendant selector.

[looks up spec]  yep.  I'll open a new issue for this.  I think any `[ ... ]` selector should be supported after `..`.